style.css

```diff
@@ -0,0 +1,151 @@
+* {
+	font-family: montserrat;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+}
+header {
+	display: flex;
+	height: 100px;
+	justify-content: center;
+}
+.container_header {
+	display: flex;
+	justify-content: space-between;
+	width: 80%;
+	padding-top: 20px;
+	margin: auto;
```

Comment:
Você teve que colocar esse `margin: auto` provavelmente porque o `display: flex` que colocou no header aqui em cima não funcionou. Ele não funcionou porque o bootstrap sobreescreveu o seu css aqui e colocou um `display: block:` no lugar do `flex`. Sempre tem que tomar cuidado com css quando aplica estilos genéricos, e eles acabam se sobreescrevendo. 

O css tem uma ordem e prioridade que ele aplica. Da uma olhada nesse texto aqui:
https://www.htmlgoodies.com/beyond/css/css-specificity-and-the-cascade.html